doc/fr/Sil/Component/Address/domain/index.rst

```diff
@@ -0,0 +1,77 @@
+Adresses
+========
+
+Ce composant est un point central pour la gestion des adresses dans le SIL.
+
+-------
+Domaine
+-------
+
+Adresse
+-------
+
+Une **Adresse** se définit par les champs suivants :
+
++-----------------+-------------------------------------------------------+--------+
+| Propriété       | Description                                           | Oblig. |
++=================+=======================================================+========+
+| street          | Numéro et voie de l'adresse                           | x      |
++-----------------+-------------------------------------------------------+--------+
+| city            | La ville                                              | x      |
++-----------------+-------------------------------------------------------+--------+
+| other           | Un complément d'adresse, nécessaire pour certains cas |        |
++-----------------+-------------------------------------------------------+--------+
+
+
+Ville
+-----
+
+Chaque **Ville** peut appartenir à un pays. On permet la saisie de ville sans forcément avoir le pays associé. À charge de chaque composant de rendre cette gestion plus ou moins stricte .
```

Comment:
But it is not actually true, the City is mandatory in the source code and in the table below.

Reply:
Yes I forgive to change this part of the doc